docs/pages/omnia3/Modeler/omnia3_modeler_components.md

```diff
@@ -0,0 +1,144 @@
+| OnLoadError       | Executed when the data Load operation is finished with error. Can be used to show the error to the user                   |
+| OnUpdateData      | Executed when the Update operation is finished with success                                                               |
+| OnUpdateDataError | Executed when the Update operation is finished with error. Can be used to show the error to the user                      |
+
+### selector
+
+The Selector component retrieves an Enumerator keys and labels translated to the user current language.
+
+This component can be used to obtain Enumerator data to be used by another element or behaviour.
+
+Component Attributes:
+
+This Component only has a configurable attribute, named _selector_. This attribute is used to define the Enumeration Selector whose configuration will be retrieved.
+
+Component Behaviours:
+
+This Component only has a Behaviour type, named _onLoad_. This behaviour is executed after Enumerator data was retrieved, and can be used to store that data to be rendered on a select Component.
+
+## 3. Modeling Components
+
+### How to add a new Components package?
+
+To add a new components package go to the Modeling area, find the **_User Interface / Components_** option on the menu and access it. This will take you to your components package management dashboard.
+
+Now select _Add new_ and fill in the following information:
+
+- **Name**: the name of the asset (needs to be unique within the model);
+- **Description**: the textual explanation of the theme's purpose (can be used as development documentation).
+- **Component file**: the package file (\*.tgz) that contains a set of components.
+
+### How to edit a Components package?
+
+By accessing **_User Interface / Components_** in the sidebar and selecting one from the list.
+
+You can upload a new version of the components package.
+
+## 4. Components Usage
+
+Components are available to be used on modeled Pages.
+
+After a Components Package is imported, components are listed on the Page _Drag to Add_ option, along with Base Components:
+
+<p align="center">
+  <img src="/images/modeler/Modeler-Components-List.jpg">
+</p>
+
+You can drag them to the Page preview area and change their Attributes, Behaviours and Styles. Each Component exposes a different set of Attributes, Behaviours and Styles that you can configure while modeling.
+
+Focusing on Attributes and Styles, its value can be set in different ways:
+
+| Value Source | Description                                                      |
+| ------------ | ---------------------------------------------------------------- |
+| Direct       | Value is set directly, with a static value                       |
+| Binding      | The dataSource instance where the entity is being created/edited |
+| Expression   | Value is computed using a Javascript code expresion              |
```

Comment:
Add a note: expressions are computed on every render of react. Must be simple operations.